bundles/tampere/content-editor/view/SideContentEditor.js

```diff
@@ -298,6 +305,82 @@ Oskari.clazz.define('Oskari.tampere.bundle.content-editor.view.SideContentEditor
             me.currentEditFeatureFid = fid;
         },
 
+        /**
+         * Edit multiple feautures
+         * @method  _editMultipleFeatures
+         * @param   {String}              fid feature identifier
+         * @private
+         */
+        _editMultipleFeatures: function (fid) {
```

Comment:
What does this mean that you edit multiple features but it only takes a single feature id as param?